docker-compose.yml

```diff
@@ -56,7 +55,16 @@ services:
     depends_on:
       - mysql
     volumes:
-      - ./:/var/www/html:delegated
+      - ./:/var/www/html
+      # If you develop on Mac or Windows you can remove the vendor/ directory
+      # from the bind-mount for better performance by enabling the next line:
+      # - /var/www/html/vendor  
+      # - /var/www/html/var
+      # - /var/www/html/admin-dev/themes/default/node_modules
+      # - /var/www/html/admin-dev/themes/new-theme/node_modules
+      # - /var/www/html/themes/node_modules
+      # - /var/www/html/themes/classic/_dev/node_modules
+      # - /var/www/html/themes/hummingbird/node_modules
```

Comment:
@tyloo I'm wondering if we loose the files when the container is deleted with a `docker compose down`. 

If that's the case, it might be interesting to mount these folder with a volume like for the database?

Reply:
I understand the reason, I had these issues when I used to be on MacOS with Docker.

My concern is about disaprrearing files when restarting containers, but if I attempt starting the containers after a fresh clone, I get weird errors.

<img width="891" height="777" alt="image" src="https://github.com/user-attachments/assets/7d48139c-72a2-4d53-b4cd-abf048ef29dd" />

With an unlimited number of `/var/www/html/vendor/autoload.php still not present wait a bit more` displayed after that.

Reply:
Removed this. Thansk for the feedback!